src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -114,20 +114,39 @@ new java.text.SimpleDateFormat java.lang.String
 new java.util.ArrayList java.util.Collection
 staticMethod java.util.Arrays toString java.lang.Object[]
 staticField java.util.Calendar AM_PM
+staticField java.util.Calendar APRIL
+staticField java.util.Calendar AUGUST
 staticField java.util.Calendar DATE
 staticField java.util.Calendar DAY_OF_MONTH
 staticField java.util.Calendar DAY_OF_WEEK
 staticField java.util.Calendar DAY_OF_WEEK_IN_MONTH
 staticField java.util.Calendar DAY_OF_YEAR
+staticField java.util.Calendar DECEMBER
 staticField java.util.Calendar DST_OFFSET
 staticField java.util.Calendar ERA
+staticField java.util.Calendar FEBRUARY
 staticField java.util.Calendar FIELD_COUNT
+staticField java.util.Calendar FRIDAY
 staticField java.util.Calendar HOUR
 staticField java.util.Calendar HOUR_OF_DAY
+staticField java.util.Calendar JANUARY
+staticField java.util.Calendar JULY
+staticField java.util.Calendar JUNE
+staticField java.util.Calendar MARCH
+staticField java.util.Calendar MAY
 staticField java.util.Calendar MILLISECOND
 staticField java.util.Calendar MINUTE
+staticField java.util.Calendar MONDAY
 staticField java.util.Calendar MONTH
+staticField java.util.Calendar NOVEMBER
+staticField java.util.Calendar OCTOBER
+staticField java.util.Calendar SATURDAY
 staticField java.util.Calendar SECOND
+staticField java.util.Calendar SEPTEMBER
+staticField java.util.Calendar SUNDAY
+staticField java.util.Calendar THURSDAY
+staticField java.util.Calendar TUESDAY
+staticField java.util.Calendar WEDNESDAY
```

Comment:
No `UNDECIMBER`? Half-joking, but maybe it makes sense to add it for consistency?

Reply:
wat. =)

Reply:
https://stackoverflow.com/questions/44716517/what-does-calendar-undecimber-do 😄

Reply:
I'm still saying wat and running away from that. =)